packages/slice-machine/src/legacy/components/Navigation/Environment.tsx

```diff
@@ -98,17 +97,10 @@ export function Environment() {
     );
   }
 
-  if (
-    isUnauthenticatedError(useEnvironmentsError) ||
-    isUnauthorizedError(useEnvironmentsError)
-  ) {
+  if (isUnauthenticatedError(useEnvironmentsError)) {
     return (
       <SideNavEnvironmentSelector
-        variant={
-          isUnauthenticatedError(useEnvironmentsError)
-            ? "unauthenticated"
-            : "unauthorized"
-        }
+        variant="unauthenticated"
```

Comment:
Do we need a variant then?

Reply:
Yes, the default value is `default`, and still also use the `offline` one above